.circleci/config.yml

```diff
@@ -109,17 +114,47 @@ jobs:
     docker:
       - image: cimg/node:18.16.0
     steps:
-      - install_dependencies
-      - run: echo "!TODO"
+      - add_ssh_keys:
+          fingerprints:
+            - '97:69:fe:70:ca:84:59:06:98:ab:9c:ec:d5:26:49:8c'
+      - checkout
+      - setup_remote_docker:
+          version: 20.10.14
+      - run:
+          name: Build App image
+          command: docker build -t kuba1212/recipeapp-docker .
+      - run:
+          name: Save built image
+          command: docker save kuba1212/recipeapp-docker | gzip > appimage.tar.gz
+      - create_env
+      - run:
+          name: Send production env
+          command: |
+            scp -o StrictHostKeyChecking=accept-new ./.env $SSH_USER@$SSH_HOST:.env
+      - run:
+          name: Send image to server
+          command: |
+            scp -o StrictHostKeyChecking=accept-new ./appimage.tar.gz $SSH_USER@$SSH_HOST:appimage.tar.gz
```

Comment:
if we send the built image to a docker hub - do we need to upload this image to the server? Shouldn't the server be able to download the image from the docker hub?

Reply:
Talked about this on Backend meeting